Cargo.toml

```diff
@@ -2,9 +2,9 @@
 members = [
     "crates/sorter-core",
     "crates/sorter-cli",
+    "python"
```

Comment:
Annoying but not sure what the alternative is. If this ends up being distributed elsewhere, users will be using the version from crates.io anyway so it won't matter, I think.